src/taipy/core/_version/_version_manager.py

```diff
@@ -52,3 +74,19 @@ def set_development_version(cls, version_number: str):
     @classmethod
     def get_development_version(cls) -> str:
         return cls._repository._get_development_version()
+
+    @classmethod
+    def replace_version_number(cls, version_number):
+        if version_number is None:
+            version_number = cls.__DEFAULT_VERSION
```

Comment:
The `"version_number"` is set to None as default in many place is because it will be set to the __DEFAULT_VERSION here.
If __DEFAULT_VERSION changes (it should not but if), we don't need to worry to much.

The reason we need `"current"`, `"development"` instead of a real version number is that I don't want to call `_VersionManager.get_current_version()` on every repositories API. This function will handle that.